crates/iota/src/iota_commands.rs

```diff
@@ -363,6 +363,8 @@ pub enum IotaCommand {
     /// Invoke Iota's move-analyzer via CLI
     #[clap(name = "analyzer", hide = true)]
     Analyzer,
+    #[cfg(feature = "gen-completions")]
+    GenerateCompletions(crate::completions::CompletionCommand),
```

Comment:
kinda bothers me that the command variant doesn't match the command type

Reply:
If anyone else wants to change this, vote here 😆 It doesn't bother me, personally.